Problem2.scala

```diff
@@ -0,0 +1,53 @@
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.functions._
+
+
+
+object Problem2 extends App{
+  
+  val sparkConf = new SparkConf()
+  sparkConf.set("spark.app.Name","my app")
+  sparkConf.set("spark.master","local[2]")
+  
+  val spark = SparkSession.builder()
+  .config(sparkConf)
+  .getOrCreate()
+  
+  val ratingsRdd = spark.sparkContext.textFile("/Users/smili/Documents/Big data -Naveena/Week 12/dataset for assignment/ratings.dat")
```

Comment:
Make path name as input file path

`val ratingsRdd = spark.sparkContext.textFile("input file path/ratings.dat")`

Reply:
file-path has been updated to <input file path>